docs/query-dsl.md

```
@@ -0,0 +1,166 @@
+- `logIndex`
+
+Some batch-style tables also have `batchIndex`.
+
+Typical pattern:
+
+1) Order by those columns descending.
+2) Add an “older-than” filter that matches the last row of the previous page.
+
+Pseudo filter for “everything older than (b, tx, log)”: (expressed as nested conjunctions)
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)